src/app/components/action-button/index.html

```diff
@@ -28,14 +56,14 @@
       propertyName="actionClick"
       isOptional="true"
     >
-      Fires when users click the action button.
+      Fires when users select the action button.
     </sky-demo-page-property>
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Action button icon properties">
     <sky-demo-page-property
       propertyName="iconType">
-      Specifies an icon from the Font Awesome library. For example, to display the <sky-code>fa-filter</sky-code> icon on the action button, set <sky-code>iconType</sky-code> to <sky-code>filter</sky-code>. This property accepts <sky-code>string</sky-code> values.
+      Specifies an icon from <a href="http://fontawesome.io/icons/">the Font Awesome library</a>. For example, to display the <sky-code>fa-filter</sky-code> icon on the action button, set <sky-code>iconType</sky-code> to <sky-code>filter</sky-code>. SKY UX supports version 4.7.0 of Font Awesome. For more information about icons in SKY UX, see <a stacheRouterLink="/design/styles/icons">the icons design pattern</a>. This property accepts <sky-code>string</sky-code> values.
```

Comment:
I know this creates work for us later if we decided to upgrade to a different version of Font Awesome, but do you it's worth it to make the link `https://fontawesome.com/v4.7.0/`?  Just curious about your thoughts on this, but I'm also okay to leave the link as is.

Reply:
Good call. I'll make that change.